examples/src/main/java/org/partiql/examples/EvaluationJavaExample.java

```diff
@@ -18,10 +18,7 @@
 import org.jetbrains.annotations.NotNull;
 import org.partiql.examples.util.Example;
 import org.partiql.lang.CompilerPipeline;
-import org.partiql.lang.eval.Bindings;
-import org.partiql.lang.eval.EvaluationSession;
-import org.partiql.lang.eval.ExprValue;
-import org.partiql.lang.eval.Expression;
+import org.partiql.lang.eval.*;
```

Comment:
Wildcard import.

EDIT -- Some of the example files got converted. Can you fix that?